packages/flutter_tools/README.md

```diff
@@ -4,5 +4,5 @@ Tools for building Flutter applications.
 
 To run the tests:
 
-- pub get
-- pub run test
+- flutter packages get
```

Comment:
I don't think these need changing for flutter_tools.  flutter_tools doesnt' depend on any flutter packages last I knew, it runs in the normal dart vm.  @danrubel would know.